frontend/src/concepts/pipelines/content/createRun/useRunFormData.ts

```diff
@@ -160,6 +200,11 @@ export const useUpdateRunType = (
   }, [setFunction, initialData]);
 };
 
+const getParams = (pipeline?: PipelineKF): RunParam[] | undefined =>
```

Comment:
I think this logic is dealt with in other spots -- we may want to look for those and make this a unified utility.